shared/macros-ansible.jinja

```diff
@@ -368,6 +369,7 @@ The macro requires following parameters:
     state: present
     dest: /etc/audit/audit.rules
     create: yes
+    mode: '0640'
```

Comment:
Just a question, why this change? Is it related to ansible-lint?

Reply:
Yep it is to make ansible-lint happy :)